core/lib/qcbor/inc/qcbor/UsefulBuf.h

```diff
@@ -0,0 +1,2506 @@
+/* =========================================================================
```

Comment:
Would it be possible to add an `SPDX-License-Identifier` tag for each source and header files in `core/lib/qcbor/` . Maybe this can be addressed in a specific commit so that commit "core: import QCBOR library" dumps the files as found in the referred source stated in the commit message, without any such local changes.

Reply:
Should I handle the addition of licence headers in a separate commit from "core: import QCBOR library"?

Reply:
Yes, please. It makes it easier for the library to be updated in the future.

Reply:
DONE